### DISCUSSION TITLE
Dec 6 Office Hours: IST event, end-to-end deployment testing, marshaling handles, devnet, ...

### DISCUSSION THREAD

**Discussion by dckc** [Agoric/agoric-sdk#8625] [Office Hours]
_zoom recording to appear_

 - [Zoom recording: Agoric Dev Office Hours Dec 6, 2023](https://agoric.zoom.us/rec/share/QqBQa8MeOye8CfUuJ7dwCZcOO_pi79LrDMsJx9LK65hf6L_7BLyMOMVnXUaIkHNp.PJGnftUJaBRnH-_f)  
  Passcode: `!S^SbpL0`

----

raw tab dump

[Agoric - Deploy Secure DeFi in JavaScript](https://agoric.com/)
[Office Hours · Agoric/agoric-sdk Wiki](https://github.com/Agoric/agoric-sdk/wiki/Office-Hours)
[New Discussion · Agoric/agoric-sdk](https://github.com/Agoric/agoric-sdk/discussions/new?category=office-hours)
[Jul 19 Office Hours: promises in terms, marshalling w/board ids, ... · Agoric/agoric-sdk · Discussion #8069](https://github.com/Agoric/agoric-sdk/discussions/8069)
[Latest topics - Agoric Community Forum](https://community.agoric.com/latest)
[Gauntlet IST Nov 2nd 2023 Non-parity Event Analysis - Inter Protocol - Agoric Community Forum](https://community.agoric.com/t/gauntlet-ist-nov-2nd-2023-non-parity-event-analysis/559)
[Smart Wallet Dapp Architecture | Documentation](https://docs.agoric.com/guides/getting-started/contract-rpc.html#specifying-offers)
[agoric-3-proposals/debugTestImage.ts at main · Agoric/agoric-3-proposals](https://github.com/Agoric/agoric-3-proposals/blob/main/debugTestImage.ts)
[How to do end-to-end test of permissioned deployment? · Agoric/agoric-sdk · Discussion #8173](https://github.com/Agoric/agoric-sdk/discussions/8173)
[Errors when running through the README process · Issue #10 · agoric-labs/dapp-game-places](https://github.com/agoric-labs/dapp-game-places/issues/10)
[Implications of ag-solo vs. smart wallet architecture on Agoric smart contracts · Agoric/agoric-sdk · Discussion #8584](https://github.com/Agoric/agoric-sdk/discussions/8584)
[SPIKE: install from npm; smart-wallet / local chain only by dckc · Pull Request #881 · Agoric/documentation](https://github.com/Agoric/documentation/pull/881)
[Starting a Basic Dapp | Documentation](https://dc-npm-only.documentation-7tp.pages.dev/guides/getting-started/#getting-support)
[agoric-labs/gimix](https://github.com/agoric-labs/gimix)

some were missing... this might duplicate some...

[Agoric - Deploy Secure DeFi in JavaScript](https://agoric.com/)
[Office Hours · Agoric/agoric-sdk Wiki](https://github.com/Agoric/agoric-sdk/wiki/Office-Hours)
[Dec 6 Office Hours: IST event, end-to-end deployment testing, marshaling handles, devnet, ... · Agoric/agoric-sdk · Discussion #8625](https://github.com/Agoric/agoric-sdk/discussions/8625)
[agoric-sdk/packages/zoe/src/contracts/gimix at dc-github-namehub · Agoric/agoric-sdk](https://github.com/Agoric/agoric-sdk/tree/dc-github-namehub/packages/zoe/src/contracts/gimix)
[feat: discord auth (WIP) by dckc · Pull Request #2 · agoric-labs/agoric-passport-express](https://github.com/agoric-labs/agoric-passport-express/pull/2)
[Jul 19 Office Hours: promises in terms, marshalling w/board ids, ... · Agoric/agoric-sdk · Discussion #8069](https://github.com/Agoric/agoric-sdk/discussions/8069)
[Latest topics - Agoric Community Forum](https://community.agoric.com/latest)
[dclaudiud/Tendermint-Chunked-Genesis-Download: Python script to download the genesis.json file from an RPC enabled Tendermint node in chunks - http://[node_ip]:26657/genesis_chunked?chunk=_](https://github.com/dclaudiud/Tendermint-Chunked-Genesis-Download/tree/main)
[Gauntlet IST Nov 2nd 2023 Non-parity Event Analysis - Inter Protocol - Agoric Community Forum](https://community.agoric.com/t/gauntlet-ist-nov-2nd-2023-non-parity-event-analysis/559)
[Smart Wallet Dapp Architecture | Documentation](https://docs.agoric.com/guides/getting-started/contract-rpc.html#specifying-offers)
[agoric-3-proposals/debugTestImage.ts at main · Agoric/agoric-3-proposals](https://github.com/Agoric/agoric-3-proposals/blob/main/debugTestImage.ts)
[How to do end-to-end test of permissioned deployment? · Agoric/agoric-sdk · Discussion #8173](https://github.com/Agoric/agoric-sdk/discussions/8173)
[Errors when running through the README process · Issue #10 · agoric-labs/dapp-game-places](https://github.com/agoric-labs/dapp-game-places/issues/10)
[Implications of ag-solo vs. smart wallet architecture on Agoric smart contracts · Agoric/agoric-sdk · Discussion #8584](https://github.com/Agoric/agoric-sdk/discussions/8584)
[SPIKE: install from npm; smart-wallet / local chain only by dckc · Pull Request #881 · Agoric/documentation](https://github.com/Agoric/documentation/pull/881)
[Starting a Basic Dapp | Documentation](https://dc-npm-only.documentation-7tp.pages.dev/guides/getting-started/#getting-support)
[agoric-labs/gimix](https://github.com/agoric-labs/gimix)

**Comment by Jorge-Lopes** [comment]
@dckc can you please share the recording of today's office hours

**Reply by dckc** [reply]
yes... I added it to the start of this thread.

**Reply by Jorge-Lopes** [reply]
Thank you very much
